common/djangoapps/student/migrations/0047_add_bio_field.py

```diff
@@ -10,7 +10,7 @@ class Migration(SchemaMigration):
     def forwards(self, orm):
         # Adding field 'UserProfile.bio'
         db.add_column('auth_userprofile', 'bio',
-                      self.gf('django.db.models.fields.TextField')(null=True, blank=True),
+                      self.gf('django.db.models.fields.CharField')(max_length=3000, null=True, blank=True),
```

Comment:
I copied this from the "gender" field migration which has both ``null=True`` and ``blank=True``. I don't know if it is really necessary.

I don't see any migrations with ``db_index=False`` but I guess I'll add it for consistency since there are some specifying ``True``.